backend/webhook/beanstalk-worker/BUILD

```diff
@@ -1,32 +1,32 @@
-# gazelle:prefix redis-worker
+# gazelle:prefix beanstalk-worker
```

Comment:
Good chance to remove the data source from the package name?

Reply:
yes!

Reply:
then I think I'd call it `consumer`